README.md

```diff
@@ -74,95 +74,33 @@ This distribution comes with the following defaults:
 
 If you're currently using the SignalFx Java Agent and want to
 migrate to the Splunk Distribution of OpenTelemetry Java,
-see [Migrate from the SignalFx Java Agent](MIGRATING.md).
+see [Migrate from the SignalFx Java Agent](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/troubleshooting/migrate-signalfx-java-agent-to-otel.html).
 
-## Getting Started
+## Requirements
 
-To get started, download the JAR for the agent's [latest
-version](https://github.com/signalfx/splunk-otel-java/releases/latest/download/splunk-otel-javaagent-all.jar):
+The agent works with Java runtimes version 8 and higher. For the full list of requirements and supported libraries and versions, see [Requirements for the Java agent](https://docs.splunk.com/Observability/gdi/get-data-in/application/java/java-otel-requirements.html) in the official Splunk documentation.
 
-```bash
-curl -L https://github.com/signalfx/splunk-otel-java/releases/latest/download/splunk-otel-javaagent-all.jar \
-     -o splunk-otel-javaagent.jar
-```
+## Get started
 
-Enable the Java agent by adding the `-javaagent` flag to the runtime JVM
-parameters and set the `service.name`. For example, if the runtime parameters
-were:
-
-```bash
-java -jar myapp.jar https://splunk.com
-```
-
-Then the runtime parameters would be updated to:
-
-```bash
-java -javaagent:./splunk-otel-javaagent.jar \
-     -Dotel.service.name=my-java-app \
-     -jar myapp.jar https://splunk.com
-```
```

Comment:
I would like to propose we keep this -- if someone directly navigates to repo or someone wants to contribute to this repo, this is useful information (I understand it means duplication).

Reply:
Yes, excellent idea. We achieve two things by keeping instructions there:

- We keep a place where the team can update the smoothest path to success
- We ensure that, should the Splunk docs be down, people can still set it up

I've brought them back in https://github.com/signalfx/splunk-otel-java/pull/373/commits/38294c3b62c982166ff4d032291d3e6022183512, taken from the current instructions that I've improved.